site/src/pages/release-notes/1.3.0.mdx

```diff
@@ -0,0 +1,231 @@
+---
+date: 2020-11-30
+---
+
+## 🌟 New features
+
+- You can now use <a href="https://unixism.net/loti/what_is_io_uring.html">io_uring</a> for efficient I/O
+  processing in Linux. #3182
+  - Specify the `-Dcom.linecorp.armeria.transportType=io_uring` JVM option to enable it.
+  - This is the early stage of development so you should be careful using the feature.
+- The metrics of requests to a <type://TransientService> are now not collected by default. #3061, #3081
+  - Access logs and service logs are not recorded as well.
+  - You should use <type://TransientServiceOption> to enable them.
+    ```java
+    TransientServiceBuilder.transientServiceOptions(TransientServiceOption.WITH_METRIC_COLLECTION,
+                                                    TransientServiceOption.WITH_SERVICE_LOGGING,
+                                                    TransientServiceOption.WITH_ACCESS_LOGGING)
```

Comment:
- This is not a real example. Use a real service.
- Could we list the services that implement `TransientService` for users' convenience.